### PR TITLE
[12.0][NEW] purchase order revision module

### PR DESCRIPTION
**Pull request by renatonlima**
Hi guys,

I created this module inspired in [sale_order_revision](https://github.com/OCA/sale-workflow/tree/11.0/sale_order_revision) module in [OCA/sale-workflow](http://github.com/OCA/sale-workflow) This module allows create versions on cancelled orders, you can click on the "New copy of Quotation" button. This will create a new revision of the quotation, with the same base number and a '-revno' suffix appended. A message is added in the chatter saying that a new revision was created.

### CODE REVIEW

File: purchase_order_revision/__init__.py [hunk 1]
```diff
@@ -0,0 +1,10 @@
+# Copyright 2019 Akretion - Renato Lima (<http://akretion.com>)
+# License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl.html).
+
+from . import models
+
+
+def populate_unrevisioned_name(cr, registry):
```
Comment (HviorForgeFlow) [inline purchase_order_revision/__init__.py:7]: for better understanding move to init_hook.py file

File: purchase_order_revision/__manifest__.py [hunk 2]
```diff
@@ -0,0 +1,20 @@
+# Copyright 2019 Akretion - Renato Lima (<http://akretion.com>)
+# License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl.html).
+
+{
+    'name': 'Purchase Order Revisions',
+    'version': "12.0.1.0.0",
+    'category': "Sale Management",
+    'author': "Akretion, "
+              "Odoo Community Association (OCA)",
+    'website': 'http://www.akretion.com',
```
Comment (HviorForgeFlow) [inline purchase_order_revision/__manifest__.py:10]: use purchase-workflow github repo URL instead

File: purchase_order_revision/models/__init__.py [hunk 3]
```diff
@@ -0,0 +1,3 @@
+
```
Comment (HviorForgeFlow) [inline purchase_order_revision/models/__init__.py:1]: Nit: Remove unnecessary blank lines

**Review by HviorForgeFlow — CHANGES_REQUESTED** [review]
Code review

Remove empty .rst files if they are not used.

**Review by marylla — APPROVED** [review]
Hi there, 
I made a functional review and it LGTM.
It works the same way as sale order revisions. So, I can create a new purchase order from a cancelled order which then gets the same number but with a suffix.

**Comment by HviorForgeFlow** [comment]
Hello @renatonlima, are you planning to finish this module? Thanks,

**Comment by marylla** [comment]
@renatonlima Can you adress the suggested changes and rebase?

**Pull request closed without merging**